### PR TITLE
fix: typo on the front page

### PR DESCRIPTION
**Pull request by iankranz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <h1>Hello World!</h1>
     <ul>
```
Comment (cndwng) [inline index.html:18]: why is this list unordered?

**Review by cndwng — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (iankranz) [reply index.html:18]: order is a lie. embrace chaos

File: index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
...
       <div><li>we love</li></div>
-      <h4>websites</h4>
+      <h4>websitez</h4>
```
Comment (anniezhang11) [inline index.html:20]: great catch! lgtm

**Review by anniezhang11 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (cndwng) [reply index.html:18]: whatever

**Review by cndwng — APPROVED** [review]

**Pull request merged**